homeassistant/components/homematic/entity.py

```diff
@@ -27,20 +30,27 @@
 class HMDevice(Entity):
     """The HomeMatic device base object."""
 
-    def __init__(self, config):
+    def __init__(
+        self,
+        config: dict[str, str],
+        entity_description: SensorEntityDescription | None = None,
```

Comment:
Use the base class from `from homeassistant.helpers.entity import Entity, EntityDescription` as type